chains/evm/listener/listener.go

```diff
@@ -74,6 +74,7 @@ func (l *EVMListener) ListenToEvents(startBlock *big.Int, domainID uint8, kvrw b
 					continue
 				}
 				for _, eventLog := range logs {
+					log.Debug().Msgf("Deposit log found from sender: %s in block: %s with  destinationDomainId: %v, resourceID: %s, depositNonce: %v", eventLog.SenderAddress, startBlock.String(), eventLog.DestinationDomainID, eventLog.ResourceID, eventLog.DepositNonce)
```

Comment:
Please check one more time for issue impl details.
You can leave these logs. But the main idea was to log Transaction details